db/mongo.js

```diff
@@ -1,73 +1,70 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable no-console */
 const mongoose = require('mongoose');
-const fakerAPI = require('./faker')
-//mongoose.connect('mongodb://localhost/fecRepo', {useNewUrlParser: true});
-//Need below for docker.
-mongoose.connect('mongodb://ec2-54-215-211-187.us-west-1.compute.amazonaws.com:27017/', {useNewUrlParser: true})
-.catch(err => console.error(err));
+const fakerAPI = require('./faker');
+// mongoose.connect('mongodb://localhost/fecRepo', {useNewUrlParser: true});
+// Need below for docker.
+mongoose.connect('mongodb://ec2-54-215-211-187.us-west-1.compute.amazonaws.com:27017/', { useNewUrlParser: true })
```

Comment:
Interesting/clever way to connect your containers that I never thought of, I used "mongodb://mongo:27017/endpoint", but had a hard time getting the containers to connect. Nice to have another idea for later